templates/tab/js/default/.webappignore

```diff
@@ -0,0 +1,17 @@
+.fx
```

Comment:
tab is not hosted by Azure Web App. I think we only need to add this file in restify bot.